deploy-board/deploy_board/templates/groups/asg_config.tmpl

```diff
@@ -105,6 +81,19 @@ $(function() {
     });
 });
 
+var asgConfigVue = new Vue({
+    el: '#autoscalingConfigId',
+    data: {
+        {% if placements %}
+        placements: {{ placements | safe }},
+        remainingCapacity: {{placements| safe }}.reduce((s, e) => s + e.capacity, 0),
+        {% else %}
+        placements: {},
+        remainingCapacity: Infinity,
```

Comment:
probably `n/a` rather than infinity

Reply:
`n/a` won't be evaluated properly in this expression `if (sizeIncrease >= this.remainingCapacity) {`